Resources/Prototypes/Entities/Objects/Weapons/Guns/Ammunition/Projectiles/colony_rifle.yml

```diff
@@ -0,0 +1,10 @@
+- type: entity
+  id: BulletAR10
+  name: bullet (7.62x51)
+  parent: BaseBullet
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Projectile
+    damage:
+      types:
+        Piercing: 56
```

Comment:
The 56 is coming from the gun's 1.4 damage multiplier, the damage is actually 40, like the base rifle bullet
Also has 5 AP, but you can just use BulletRifle10x24mm as the projectile and delete this since they are the same